tt_eager/tt_lib/csrc/tt_lib_bindings_tensor_dm_ops.cpp

```diff
@@ -61,7 +61,7 @@ namespace tt::tt_metal::detail{
 
                 Input tensors must be on device, in ROW MAJOR or TILE layout, and have matching data type.
 
-                Datatype must be one ofthe following types BFLOAT16,BFLOAT8_B,BFLOAT4_B,UINT32,INT32 and UINT16.
+                Datatype must be one ofthe following types BFLOAT16,BFLOAT8_B,BFLOAT4_B,UINT32,INT32,UINT16 and UINT8.
```

Comment:
```suggestion
                Datatype must be one of the following types BFLOAT16, BFLOAT8_B, BFLOAT4_B, INT32, UINT32, UINT16 or UINT8.
```

Reply:
```C++
Datatype must be one of the following types BFLOAT16, BFLOAT8_B, BFLOAT4_B, UINT32, INT32, UINT16 and UINT8.
```
As you suggested, I have made the changes.